companion/src/radiodata.cpp

```diff
@@ -31,14 +31,21 @@ RawSource RawSource::convert(Board::Type before, Board::Type after)
   }
 
   if (IS_HORUS(after)) {
-    if (IS_TARANIS_X9D(before) || IS_TARANIS_X7(before)) {
+    if (IS_TARANIS_X9D(before)) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS1066)